client/components/Header.jsx

```diff
@@ -2,16 +2,28 @@ import React from 'react'
 import {connect} from 'react-redux'
 import {Link} from 'react-router-dom'
 
-const Header = () => {
+const Header = (props) => {
   return (
     <div className='header'>
       <Link to='/'><h1>Pings</h1></Link>
-      <ul>
+      {!props.currentUser
+        ? <ul>
+        <li><Link to='/signin'><strong>Sign In</strong></Link></li>
+        <li><Link to='/register'><strong>Register</strong></Link></li>
+      </ul>
```

Comment:
indent line

Reply:
Since you don't have an "else" in your ternary, it would be better to use `&&`